flight/targets/quanton/fw/pios_board.c

```diff
@@ -624,7 +624,7 @@ void PIOS_Board_Init(void) {
 		}
 
 		if (PIOS_I2C_CheckClear(pios_i2c_usart1_adapter_id) != 0)
-			panic(6);
```

Comment:
I just tested again with a attiny as a i2c slave and also with a an external mag (HMC5883 GY-273).
If unpowered they pull the bus down and checkclear fails.